src/styles/blocks/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be consistent indentation. Ensure that the properties are properly indented with 2 spaces.